tools/modules/AksEdgeDeploy/AksEdgeDeploy.psm1

```diff
@@ -914,7 +914,9 @@ function Invoke-AideDeployment {
     $retval = New-AksEdgeDeployment -JsonConfigString $aksedgeDeployParams
 
     if ($retval -ieq "OK") {
-        Write-Host "* AksEdge VM deployment successfull." -ForegroundColor Green
+        Write-Host "* AksEdge VM deployment successful." -ForegroundColor Green
+    } elseif ($retval -ieq "arc parameters not set or invalid") {
+        Write-Host "* AksEdge VM deployment successful but arc connection failed." -ForegroundColor Green

```

Comment:
set $retval to "OK" and remove the changes to AksEdgeQuickStart.ps1 & AksEdgeQuickStartForAio.ps1